### PR TITLE
atomic: fix some portabilities issues

### PR DESCRIPTION
**Pull request by giuseppe**
Patches needed to run under Ubuntu.

Related to: https://github.com/projectatomic/atomic/issues/1165

After I've installed Skopeo and runc, I've also been able to run system containers on Ubuntu 17.10.

### CODE REVIEW

File: Atomic/util.py [hunk 1]
```diff
@@ -36,8 +36,8 @@
 ATOMIC_INSTALL_JSON = os.environ.get('ATOMIC_INSTALL_JSON', os.path.join(ATOMIC_VAR_LIB, 'install.json'))
 
 GOMTREE_PATH = os.environ.get("GOMTREE_PATH", "/usr/bin/gomtree")
-BWRAP_OCI_PATH = os.environ.get("BWRAP_OCI", "/usr/bin/bwrap-oci")
-RUNC_PATH = os.environ.get("RUNC", "/bin/runc")
+BWRAP_OCI_PATH = os.environ.get("BWRAP_OCI_PATH", "/usr/bin/bwrap-oci")
+RUNC_PATH = os.environ.get("RUNC_PATH", "/usr/bin/runc")
```
Comment (baude) [inline Atomic/util.py]: I think ubuntu's runc ends up in /usr/sbin now ... do you care?

Comment (giuseppe) [reply Atomic/util.py]: yes, by default it doesn't work, but we can't make happy both Fedora and Ubuntu :-( so I picked Fedora and expect RUNC_PATH to be specified on Ubuntu

**Comment by baude** [comment]
LGTM one little question

**Comment by baude** [comment]
@rh-atomic-bot r+

**Comment by giuseppe** [comment]
@rh-atomic-bot r=baude 8e344de

**Pull request closed without merging**